pkg/cli/cmds/server.go

```diff
@@ -143,6 +143,10 @@ func ServerRun(clx *cli.Context) error {
 
 func validateCNI(clx *cli.Context) {
 	cni := clx.String("cni")
+	if !strings.Contains(cni, "multus") {
+		clx.Set("disable", "rke2-multus")
+	}
+	cni = strings.Split(cni, "+")[0]
```

Comment:
Lets be a little more picky about the suffix
```suggestion
	if strings.HasSuffix(cni, "+multus") {
		cni = strings.TrimSuffix(cni, "+multus")
	} else {
		clx.Set("disable", "rke2-multus")
	}
```